linkerd/tox.ini

```diff
@@ -0,0 +1,22 @@
+[tox]
+minversion = 2.0
+basepython = py27
+envlist =
+    linkerd
+    flake8
+
+[testenv]
+platform = linux2|darwin
+
+[testenv:linkerd]
+deps =
+    ../datadog-checks-base
+    -rtests/requirements.txt
```

Comment:
Does this work for you? I had to do `-r./tests/requirements.txt`.

Reply:
both work for me. I will still change it to `-r./tests/requirements.txt` since it work on both our envs